src/openrct2/ride/Track.h

```diff
@@ -49,9 +49,106 @@ enum class TrackPitch : uint8_t
     ReverseFreefall = 10
 };
 
+// Vehicle sprite groups required by track groups are defined in ride_entry_get_supported_track_pieces
+enum TrackGroup : uint8_t
+{
+    flat = 0,
```

Comment:
UpperCamelCase?

Reply:
No. We changed it earlier this year to follow OpenLoco’s code style.